envs/local.env

```diff
@@ -5,6 +5,13 @@ APP_ENV = prod
 GRPC_HOST=localhost
 GRPC_PORT=3000
 
+# HTTP
+HTTP_PORT=4000
+
+# GRPC clients
+GO_TEMPLATE_GRPC_PORT=3000
```

Comment:
Let's just have `HTTP_PORT` and `GRPC_PORT`.
It does not related to project name.

Reply:
Sure, but this is for the external client that has been used in external layer.

Reply:
@SamanNsr , So Its like `EXTERNAL_SERVICE_PORT` ?
 Thats Ok

Reply:
Maybe `EXTERNAL_SERVICE_PORT` is better convention!

Reply:
@SamanNsr Thats ok for Template.
But Lets add `EXTERNAL` keyword here